### ISSUE TITLE
Large performance gap between my trained VAE and the released pretrained model in VecSetX

### ISSUE THREAD

**Issue by muradilkurban** [1zb/VecSetX#7]
I’m training the VAE from VecSetX but the reconstruction quality is bad. Here’s what I did:

1. Data: Used the ShapeNet car GLB files → extracted SDF/point clouds with sdf_gen/preprocess/box.py → saved as .npz. Visualized samples, data looks fine.
2. Model: Used learnable_vec1024x16_dim1024_depth24 (no changes).
3. Training: Ran vecset/main_ae.py with 2 GPUs + gradient accumulation (8 steps). Trained for ~500 epochs.
4. Results: Final loss ~0.23, volume IoU ~0.64.
5. Inference: Used vecset/infer.py at 128/256 resolution. Output looks poor.

Any ideas why the quality is low? Thanks! 😊

<img width="782" height="536" alt="Image" src="https://github.com/user-attachments/assets/198994cd-fdc3-41e0-b63a-0d462d73d634" />
